verification/src/contextual_block_verifier.rs

```diff
@@ -550,19 +554,13 @@ impl<'a, CS: ChainStore<'a>> ContextualBlockVerifier<'a, CS> {
             RewardVerifier::new(&self.context, resolved, &parent).verify()?;
         }
 
-        let cycles = if !switch.disable_txs() {
-            BlockTxsVerifier::new(
-                &self.context,
-                block.number(),
-                block.epoch(),
-                parent_hash,
-                resolved,
-            )
-            .verify(txs_verify_cache, executor)?
-        } else {
-            0
-        };
-
-        Ok(cycles)
+        BlockTxsVerifier::new(
```

Comment:
rebase error? I think that `switch` is introduced in https://github.com/nervosnetwork/ckb/pull/1643

Reply:
No, since we put the tx fee in the verifier cache, the `disable_txs` is no-sense, the `dao` will not calculate correctly if we `disable_txs`. furthermore I don't see this flag enabled anywhere.

Reply:
IMO we can add it back when we really need the flag.